build.gradle

```diff
@@ -130,6 +130,9 @@ dependencies {
     api platform('com.exactpro.th2:bom:2.3.+')
     api 'com.exactpro.th2:cradle-core'
     api 'com.exactpro.th2:grpc-common'
+    
+    //FIXME: Move to grpc-common
+    implementation "com.exactpro.th2:service-generator-protoc:3.0.0"
 
     implementation 'com.google.protobuf:protobuf-java-util'
     implementation 'com.exactpro.th2:grpc-service-generator'
```

Comment:
It is the same dependency. We should use it instead of the dependency in line 135 or remove it